src/Uno.UI/FeatureConfiguration.cs

```diff
@@ -92,14 +92,24 @@ public static class ComboBox
 
 		public static class CompositionTarget
 		{
+			private static float _frameRate = 60;
+
 			/// <summary>
-			/// The delay between invocations of the <see cref="Microsoft.UI.Xaml.Media.CompositionTarget.Rendering"/> event, in milliseconds.
-			/// Lower values will increase the rate at which the event fires, at the expense of increased CPU usage.
-			///
-			/// This property is only used on WebAssembly.
+			/// Suggested frame rate for <see cref="Microsoft.UI.Xaml.Media.CompositionTarget.Rendering"/> event.
+			/// This property is used by desktop skia renderers.
 			/// </summary>
-			/// <remarks>The <see cref="Microsoft.UI.Xaml.Media.CompositionTarget.Rendering"/> event is used by Xamarin.Forms for WebAssembly for XF animations.</remarks>
-			public static int RenderEventThrottle { get; set; } = 30;
+			public static float FrameRate
```

Comment:
Why are there two feature flags for the same purpose (`FrameDuration` and `FrameRate`)? Probably I'm just missing something

Reply:
This is essentially the same value but at different layers, will adjust. The dispatching one is internal.